ansible/config/hosts

```diff
@@ -29,8 +20,11 @@ dcc-hadoop-master
 [hdfs-datanode]
 dcc-hadoop-worker-[1:2]
 
-[spark-master]
-dcc-hadoop-worker-1
+[mongo]
+dcc-mongo
+
+[id-group]
```

Comment:
Maybe we should start naming things like `dcc-id`, `dcc-release`, `dcc-submission`?